src/assets/traffic-icons/Close.svg.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
```

Comment:
hehehe we don't need to import React in this codebase when used as type. No issues though :)

Reply:
That's : https://react-svgr.com/playground/?typescript=true
👀